routing/posts/postRoutes.js

```diff
@@ -0,0 +1,94 @@
+/* --- Routes for '/api/posts' --- */
+const express = require('express'); // import the express package
+
+const db = require('../../data/db.js'); // import data helpers
+
+const router = express.Router(); // creates the route
+
+
+// POST - Creates a post using info sent in request body
+router.post('/', (req, res) => {
+  const { title, contents } = req.body;
+  const post =  { title, contents }
+  if (!title || !contents) {
+    res.status(400).json({ error: 'Please provide title and contents for the post.' })
+  }
+  db
+  .insert(post)
+  .then(postID => {
+    res.status(201).json(postID);
```

Comment:
Nice use of 201, a subtle difference between 200, but it's the correct status to use.